Formula/digdag.rb

```diff
@@ -1,9 +1,8 @@
 class Digdag < Formula
   desc "Workload Automation System"
   homepage "https://www.digdag.io/"
-  url "https://dl.digdag.io/digdag-0.9.42.jar"
-  sha256 "71071424a5beeb881cb4ccb06067a556a3673caf3dc8850dd584357e99134b21"
-  revision 1
+  url "https://dl.digdag.io/digdag-0.10.0.jar"
+  sha256 "0a3aed836d8af1a47ed53dda63c02ce3ecfec6b564d55b556a18b122dec7f3d7"
```

Comment:
```suggestion
  sha256 "0a3aed836d8af1a47ed53dda63c02ce3ecfec6b564d55b556a18b122dec7f3d7"
  license "Apache-2.0"
```
https://github.com/treasure-data/digdag/blob/master/LICENSE